src/user/user.spec.js

```diff
@@ -55,6 +55,27 @@ describe("user", () => {
     });
   });
 
+  describe(".getAll() :", () => {
+    it("gets all the users created", async () => {
+      const userId1 = await xo.createUser({
+        email: "wayne4@vates.fr",
+        password: "batman",
+        permission: "user",
+      });
+      const userId2 = await xo.createUser({
+        email: "wayne5@vates.fr",
+        password: "batman",
+        permission: "user",
+      });
+      const users = await xo.call("user.getAll");
+      expect(Array.isArray(users)).toBe(true);
+      expect([
+        omit(find(users, { id: userId1 }), "id"),
```

Comment:
Why not just omitting the ids from users and simply test them with their snapshot ?

Reply:
We use the `id` to find the informations of the users created before in `users` : 
`... omit(find(users, { id: userId1 }), "id") ...`

Do you want to change the parameter of `find` (for example :  find by 'email') ?